scripts/us_census/pep/us_pep_sexrace/preprocess.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This script runs
+all the national state and
+county python script and generate
+three output csv, mcf and tmcf.
+"""
+
+import os
+import json
+from absl import app
+from absl import flags
+import enum
+from national.national_1900_1970 import process_national_1900_1970
+from national.national_1980_1990 import process_national_1980_1990
+from national.national_1990_2000 import process_national_1990_2000
+from national.national_2000_2010 import process_national_2000_2010
+from national.national_2010_2020 import process_national_2010_2020
+from state.state_1970_1979 import process_state_1970_1979
+from state.state_1980_1990 import process_state_1980_1990
+from state.state_1990_2000 import process_state_1990_2000
+from state.state_2000_2010 import process_state_2000_2010
+from state.state_2010_2020 import process_state_2010_2020
+from county.county_1970_1979 import process_county_1970_1979
+from county.county_1980_1989 import process_county_1980_1989
+from county.county_1990_2000 import process_county_1990_2000
+from county.county_2000_2009 import process_county_2000_2009
+from county.county_2010_2020 import process_county_2010_2020
+from postprocess import create_single_csv, generate_mcf, generate_tmcf
+
+
+class Outputfiles(enum.Enum):
```

Comment:
This has already been defined in postprocess.py?

In general, don't define the same thing repeatedly, if such a thing is used in multiple files, maybe add a util lib to define these things.

Reply:
Abstracted the common class used and placed it in common file.